### PR TITLE
remove django history from course

### PR DESCRIPTION
**Pull request by ahsan-ul-haq**
LEARNER-3348

### CODE REVIEW

File: ecommerce/courses/management/commands/sync_history_with_course.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                course.save()
+            except ObjectDoesNotExist:
+                logger.warning(
+                    'History object for course with course_id: %s does not exist',
+                    course.id
+                )
+                course.created = None
+                course.modified = None
+                course.save()
+                continue
```
Comment (waheedahmed) [inline ecommerce/courses/management/commands/sync_history_with_course.py]: No need for this `continue`, remove it.

File: ecommerce/courses/management/commands/sync_history_with_course.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = 'Sync course history data with course created modified date.'
+
+    def handle(self, *args, **options):
+        courses = Course.objects.all()
+
+        for course in courses:
```
Comment (waheedahmed) [inline ecommerce/courses/management/commands/sync_history_with_course.py]: Do we need to do this in atomic block?

**Review by waheedahmed — APPROVED** [review]
Couple of comments otherwise looks good.

**Comment by ahsan-ul-haq** [comment]
@mjfrey please review.

**Comment by mjfrey** [comment]
What is the impact of running these migrations in production?  Will this take a long time?  We need to prepare devops if so and maybe break this up into 2.

**Comment by ahsan-ul-haq** [comment]
It will not take long time. As I don't think number of courses are more 3500.

**Review by mjfrey — APPROVED** [review]
Please squash commits.

**Pull request closed without merging**